src/containers/NdlaFilm/components/DropdownSearch.jsx

```diff
@@ -12,42 +12,63 @@ import { AsyncDropdown } from '../../../components/Dropdown';
 import { searchResources } from '../../../modules/search/searchApi';
 import config from '../../../config';
 
-const queryResources = async input => {
-  const query = {
-    page: 1,
-    subjects: 'urn:subject:20',
-    'context-types': config.ndlaFilmArticleType,
-    sort: '-relevance',
-    'page-size': 10,
-    query: input,
+const DropdownSearch = ({
+  selectedElements,
+  placeholder,
+  onChange,
+  subjectId,
+  clearInputField,
+  onClick,
+}) => {
+  const queryResources = async input => {
+    let query = {
+      page: 1,
+      subjects: `urn:subject:${subjectId || '20'}`,
+      sort: '-relevance',
+      'page-size': 10,
+      query: input,
+    };
+    subjectId
+      ? (query = { ...query })
+      : (query = {
+          ...query,
+          'context-types': config.ndlaFilmArticleType,
+        });
```

Comment:
Kan forenkles litt
https://stackoverflow.com/questions/47892127/succinct-concise-syntax-for-optional-object-keys-in-es6-es7
```suggestion
  const queryResources = async input => {
    const query = {
      page: 1,
      subjects: `urn:subject:${subjectId || '20'}`,
      sort: '-relevance',
      'page-size': 10,
      query: input,
      ...(subjectId && { 'context-types': config.ndlaFilmArticleType }),
    };
```